ast/check.go

```diff
@@ -235,23 +235,28 @@ func (tc *typeChecker) checkRule(env *TypeEnv, as *AnnotationSet, rule *Rule) {
 		tpe = types.Or(exist, f)
 
 	} else {
-		switch rule.Head.DocKind() {
-		case CompleteDoc:
+		switch rule.Head.RuleKind() {
```

Comment:
ℹ️ We're moving the DocKind: Complete/PartialObject/PartialSet to RuleKind: SingleValue/MultiValue. It's mostly a cosmetics thing right now; or rather, we're dealing with this upfront instead of later.